ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -872,6 +889,14 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
                                                 <activeExtension.flyout application={application} tree={tree} />
```

Comment:
would also suggest making the above two lines identical to the new ones or vice-versa for consistency.

(could potentially consolidate these two panels as well, since it looks like `isMiddle` is the only different prop right now and it is optional for the panel, but these two extensions could expand with different APIs, so I wouldn't necessarily suggest that just yet)